third_party/openssl/BUILD.openssl.bazel

```diff
@@ -38,8 +38,8 @@ configure_make(
     }),
     lib_source = ":all_srcs",
     out_static_libs = [
```

Comment:
The ordering here is apparently significant. Without reordering, rust linking fails. I'm not sure if that's an issue in some recent version of rules_rust or what.